todo_app/data/classes.py

```diff
@@ -15,4 +15,32 @@ def __init__(self, id, name):
 
     @classmethod
     def from_trello_list(cls, list):
-        return cls(list['id'], list['name'])
\ No newline at end of file
+        return cls(list['id'], list['name'])
+
+class ViewModel:
+    def __init__(self, items, lists):
+        self._items = items
+        self._lists = lists
+
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def lists(self):
+        return self._lists
+
+    @property
+    def to_do_items(self):
+        list_id = next((list.id for list in self._lists if list.name == 'To Do'), None)
+        return [item for item in self._items if item.list_id == list_id]

```

Comment:
Minor but I'd be inclined to see if we can strip out this logic based on lists; the ViewModel should be shielded from Trello's implementation and it feels like if we updated our Item class to store both list-name (possibly as `status` and `listId` (if we want/need it)) then I think we could reduce the knowledge ViewModel needs to have about the underlying lists?

Reply:
That said, it would probably require switching endpoints to the lists one with the `open` parameter so that we get all the info we need in one place